### DISCUSSION TITLE
[Android] How to convert .mat file to .filament file

### DISCUSSION THREAD

**Discussion by man4soo** [google/filament#4165] [Q&A]
Hi there.
I'm new to the filament library. Based on Android example codes, I found that I should convert my mat file to a filament file.
Can you please let me know how to convert a mat file to a filament file on Android?
Thanks.

**Comment by romainguy** [comment]
You can do this by using the tool called "matc" (material compiler). Please check the documentation at https://google.github.io/filament/Materials.html#compilingmaterials
